prebid-server/developers/add-new-bidder-go.md

```diff
@@ -673,8 +672,7 @@ func buildMeta(bid *adapters.TypedBid) (json.RawMessage, error) {
       AgencyName:           "Some Agency Name",
       AdvertiserID:         3,
       AdvertiserName:       "Some Advertiser Name",
-      AdvertiserDomains:    bid.ADomain,
-      dchain:               bid.ext.dchain,      
+      DChain:               DEMAND_CHAIN_JSON,
```

Comment:
Huh. I never actually coded up the DChain. I'll do that soon. I think it's fine to leave in for now - no one's using it anyway. I'd like to include a proper syntax example, so instead of `DEMAND_CHAIN_JSON` let's use ``json.RawMessage(`{Some Demand Chain JSON}`),``

Reply:
updated